wolfcrypt/src/asn.c

```diff
@@ -13878,6 +13878,29 @@ static const ASNItem altNameASN[] = {
 #define altNameASN_Length (sizeof(altNameASN) / sizeof(ASNItem))
 #endif /* WOLFSSL_ASN_TEMPLATE */
 
+
+static void AddAltName(DecodedCert* cert, DNS_entry* dnsEntry)
+{
+#ifdef OPENSSL_EXTRA
```

Comment:
Can you also add a new macro check like `!defined(WOLFSSL_ALT_NAMES_NO_REV)`, to allow people the option to keep existing behavior?